packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -157,9 +161,18 @@ void main() {
     expect(tester.getRect(find.text('Hint')).size, equals(const Size(800.0, 16.0)));
     expect(tester.getRect(find.text('Hint')).left, equals(0.0));
     expect(tester.getRect(find.text('Hint')).top, equals(294.5));
-    expect(tester.getRect(find.text('Helper')).size, equals(const Size(800.0, 12.0)));
+    expect(
```

Comment:
This is strange (here and below..). A comment that explains why we're allowing the width to vary by 1 pixel?

Reply:
There is a TODO at the top of this block of expects that explains this, with a bug number.